go/vt/vtgate/engine/delete.go

```diff
@@ -91,7 +91,7 @@ func (del *Delete) deleteVindexEntries(ctx context.Context, vcursor VCursor, bin
 	for i := range rss {
 		queries[i] = &querypb.BoundQuery{Sql: del.OwnedVindexQuery, BindVariables: bindVars}
 	}
-	subQueryResults, errors := vcursor.ExecuteMultiShard(ctx, rss, queries, false /* rollbackOnError */, false /* canAutocommit */)
+	subQueryResults, errors := vcursor.ExecuteMultiShard(ctx, del, rss, queries, false, false)
```

Comment:
nit: can we keep the comments for the bool passed, easier to know what those `false, false` mean.

Long term we could have const that can be used instead of `false` directly.

Reply:
If you agree, then there are other instances as well.

Reply:
Fxied